src/dynapath/util.clj

```diff
@@ -21,11 +22,21 @@
   (if (readable-classpath? cl)
     (dc/classpath-urls cl)))
 
+(defn- system-classpath
+  "Starting with Java 9, the default class loader is no longer an
+  instance of URLClassLoader, so `classpath` returned an empty sequence.
+  See more in clojure/java.classpath"
+  []
+  (map #(File. ^String %)
```

Comment:
I believe this should return `URL`s, not `File`s, since it will be returned by `all-classpath-urls`.

Reply:
Addressed this by `(-> (File. ^String %) .toURI .toURL)` just because `java.class.path` can contain `src` or smth like that and need to be resolved